### DISCUSSION TITLE
How to get a dirsparity map by non default?

### DISCUSSION THREAD

**Discussion by BigBigFlyPig** [microsoft/AirSim#3309] [Support Q&A]
I am using the CV mode and i have read the code about CV_pawns setting,i found the default offset between left_camera to right_camera is (0,2*12.5,0),i want to set the camera position (offset) by my self and  get the correct disparity map,but i was confused about the way that i get a disparity map,(i guess the disparity map is based on the depth map and some default settings like focal length),and how could i get the disparity map correctly when i set the offset by myself?

**Comment by KailinTong** [comment]
Hi, 

I am facing the same problem.
do you have any updates about this question?

Thanks a lot!
